Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+#!groovy
+
+properties([
+        buildDiscarder(logRotator(numToKeepStr: '10', artifactNumToKeepStr: '2')),
+])
+
+node('linux') {
```

Comment:
If this should really only run for PRs then I suggest wrapping this whole thing in a `if (env.CHANGE_ID) {`